helm/azure-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: azure-operator
+  namespace: giantswarm
+rules:
+  - apiGroups:
+      - "provider.giantswarm.io"
+    resources:
+      - azureconfigs
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - create
+      - delete
```

Comment:
Do we need namespaces at all?

Reply:
I didn't mean namespace in `metadata:` JFYI. I think it was good, the deployment is in the giantswarm namespace.

Reply:
I removed namespace from metadata because it is a cluster role and it does not need. About the resource namespace if the azure operator does not need it at all then I remove it.

Reply:
indeed if azure operator won't use namespaces then we can do a role binding instead of cluster role binding for the cluster role, so it would be more restricted